src/clojure/nrepl/middleware/pr_values.clj

```diff
@@ -1,43 +1,67 @@
 (ns nrepl.middleware.pr-values
   {:author "Chas Emerick"}
   (:require
-   [nrepl.middleware :refer [set-descriptor!]])
+    [clojure.string :as str]
+    [nrepl.middleware :as middleware])
   (:import
-   nrepl.transport.Transport))
+    nrepl.transport.Transport))
+
+
+(defn- print-renderer
```

Comment:
I guess a better name for this might be `default-print-renderer` or something along those lines.